.github/workflows/haskell.yml

```diff
@@ -47,97 +47,111 @@ jobs:
       DOCTEST: ${{ matrix.doctest }}
 
     runs-on: ${{ matrix.runner }}
+    continue-on-error: ${{ matrix.ignore_error }}
     strategy:
-      fail-fast: false
+      fail-fast: true
       matrix:
-        name:
-          - 9.2.1
-          - 9.0.1
-          - 8.10.7+stack
-          - 8.10.7+macOS
-          - 8.10.7+linux+coverage+cabal
-          - 8.10.7+docspec
-          - 8.8.4
-          - 8.8.4+inspection+fusion-plugin+Werror
-          - 8.6.5+fusion-plugin
-          - 8.6.5+streamk
-          - 8.4.4+debug
+        # The order is important to optimize fail-fast.
+        variant:
+          - 8.10.7-sdist-fusion-inspection-Werror
+          - 9.2.1-docspec
+          - 8.10.7-coverage
```

Comment:
It would be nice to have all the `variants` to be listed here. More informative and easier to read through.

Reply:
Or none of them. Does `include` not respect the order?